phantom-tester.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+var phantom = require('phantom')
+// config file containing the repl keys
+// for development, create a config file exporting the following :
+// module.exports = { token : { msg_mac: 'key', time_created : timestamp}}
+var config = require('./config/config')
+
+var _ph, _page
+
+phantom.create().then(function (ph) {
+  _ph = ph
+  return _ph.createPage()
+}).then(function (page) {
+  _page = page
+  // print all console messages in the webpage
+  _page.property('onConsoleMessage', function (msg) {
+    console.log('CONSOLE: ' + msg)
+  })
+  // print all the error messages in the webpage
+  _page.property('onError', function (msg) {
+    console.log('phantomjs page error : ' + msg)
+  })
+  /*
+  _page.property('onResourceReceived', function (msg) {
+    console.log('received : ' + msg)
+  })
+  */
+  _page.property('onCallback', function (data) {
+    console.log('Received response : ' + data.payload)
+  })
+  return _page
+}).then(function (page) {
+  // run the tester.html page in a separate http server (python -m SimpleHTTPServer 9001)
+  // will be later incorporated in this module itself
+  return _page.open('http://localhost:9001/tester.html')
```

Comment:
The localhost server is running on 9001. But better parameterize it, so that others can run this test suite from their machine.

Same with server URL too.